d2l-table.html

```diff
@@ -168,7 +168,7 @@
 			properties: {
 				role: {
 					type: String,
-					value: 'table',
+					value: 'grid',
```

Comment:
Hmm, don't think we want this change in isolation. The spec says it's supposed to be used only for tabular *data*, whereas tables can be used for non-data things. It also requires that cells use the `gridcell`.

Reply:
d2l-table-wrapper can be used for tabular data. `d2l-*` elements are needed mostly for dynamic tables.

See https://github.com/Brightspace/d2l-rubric/pull/32. The aria tests were failing because table is not recognized as a valid role.

Reply:
Gotcha, I see now that `table` is an invalid role. But I think if we're changing this to `grid`, then `d2l-td` needs to be of role `gridcell`. There seems like more work we need to do here around row groups (thead/tbody/tfoot) and well as row and column header roles. [The spec](https://www.w3.org/WAI/PF/aria/roles#grid)] has all the details.

Sounds like some screen reader testing would be good too.